build-tools/packages/build-cli/src/commands/info.ts

```diff
@@ -43,34 +95,53 @@ export default class InfoCommand extends BaseCommand<typeof InfoCommand> {
 				? context.packagesInReleaseGroup(flags.releaseGroup)
 				: [...context.fullPackageMap.values()];
 
+		// Sort by packages by name (invariant sort by codepoints).
+		// (See https://stackoverflow.com/a/40355107)
+		packages.sort(
+			(left, right) => Number(left.name > right.name) || -(left.name < right.name),
+		);
```

Comment:
This replaces the use of 'sort-package-json' at the end, which was no longer sorting since I return a JSON table (array of objects) rather than a map-like object.